scripts/update-rpc-bindings.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+# you need to have dcrpcgen
+
+# generate schema.json
+ROOT_DIR=$PWD
+cd ./jni/deltachat-core-rust/deltachat-rpc-server
+cargo run -- --openrpc > "$ROOT_DIR/schema.json"
+cd "$ROOT_DIR"
+
+# generate code
+dcrpcgen java --schema schema.json -o ./src/main/java/
```

Comment:
Somehow this file has no trailing newline, you might want to fix your editor config.